templates/public-adventures/body.html

```diff
@@ -11,7 +11,7 @@
                 value="{{currentSearch}}"
                 {% if currentSearch %}autofocus{% endif %}
                 hx-trigger="input changed delay:500ms, search" 
-                hx-post="/public-adventures/filter?level={{level}}&lang={{selectedLang}}&tag={{selectedTag}}&search={{currentSearch}}"
+                hx-put="/public-adventures/filter?level={{level}}&lang={{selectedLang}}&tag={{selectedTag}}&search={{currentSearch}}"
```

Comment:
`PUT` is an odd verb to use for this operation. Normally for read-only operations, we would use `GET`, or perhaps `POST`.

Why did you change it?